test/CoreTest.cpp

```diff
@@ -688,14 +688,14 @@ TEST_CASE( "structure_test", "[core]" )
 
 TEST_CASE( "wrapper_test", "[core]" )
 {
-    Series o = Series("./new_openpmd_output.json", Access::CREATE);
+    Series o = Series("./new_openpmd_output_%T.json", Access::CREATE);
 
     o.setOpenPMDextension(42);
     o.setIterationEncoding(IterationEncoding::fileBased);
```

Comment:
Fun fact: This is exactly the kind of wrong usage that #1048 reports. Since this PR adds sanity-checks to catch such behavior, I'm changing this Series to be file-based now.